assignment2.py

```diff
@@ -0,0 +1,44 @@
+def blank_check(check):
+    while check == '':
+        check = input("Please enter the required information: ")
+    else:
+        return check
+
+
+
+name = input("what is your name?: ")
+name = blank_check(name)
+print(f"Ok, {name}! Let's gather a few more details about your stay: ")
+conf_id = input("what is your conference ID?: ")
+conf_id = blank_check(conf_id)
+org = input("what organization do you represent?: ")
+org = blank_check(org)
+email = input("what is your email address?: ")
+email = blank_check(email)
+food = input("any food preferences?: ")
+food = blank_check(food)
```

Comment:
here is a bit better approach:
```

def get_user_input_for(ask):
    response = None
    while not response:
        response = input(ask).strip()
    return response


name = get_user_input_for("what is your name?: ")
conf_id = get_user_input_for("what is your conference ID?: ")

```

some benefits:
* logic is contained in a single function, right now you have the same logic in two different places
* you're not renaming the variable with the blank_check
* code is easier to read and follow